CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ in order to get warned about deprecated features used in your code.
 
 This can also be enabled programmatically with `warnings.simplefilter('default', DeprecationWarning)`.
 
+## [2.6.2] - not released yet
+### Added
+* `skew` function (similar to `rotation`), along with relevant unit tests and documentation.
```

Comment:
```suggestion
* [`FPDF.skew()`](https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.skew) - Documentation: [documentation page](https://pyfpdf.github.io/fpdf2/Transformations.html) - thanks to @erap129
```
Also, could you add this in the section dedicated to the upcoming version `2.6.1` please?

Reply:
done